ee/clickhouse/queries/funnels/test/test_funnel_trends.py

```diff
@@ -93,8 +93,11 @@ def test_no_event_in_period(self):
 
         funnel_trends = ClickhouseFunnelTrends(filter, self.team, ClickhouseFunnel)
         results = funnel_trends._exec_query()
+        formatted_results = funnel_trends._format_results(funnel_trends._summarize_data(results))
 
         self.assertEqual(len(results), 7)
+        print(formatted_results)
```

Comment:
FYI stale print statement

Reply:
It actually looks like the test is WIP so let me know when we can take a look

Reply:
heh yeah, how do I run clickhouse backend tests again? 😅 Do I need clickhouse locally?

Reply:
Yes, usually I'll run clickhouse, zookeeper & kafka with Docker compose and run the tests in the virtualenv. If you want the print to show you need to add the `-s` flag to the `pytest` command.

Reply:
ok it's ready for review!!